### PR TITLE
Fix grid tile coordinates calculation

### LINKED ISSUE #184: Biggest tissue box coordinates are inverted (PIL -> NumPy)
**Describe the bug**
Tiler's area of extraction is wrong, i.e. the biggest tissue box coordinates (W,H) are inverted (H,W)

**To Reproduce**

```python
from histolab.data import prostate_tissue
from histolab.slide import Slide
from histolab.tiler import GridTiler

f, path = prostate_tissue()
prostate_slide = Slide(path, 'proc')

grid_tiles_extractor = GridTiler(
   tile_size=(400, 200),
   level=0,
   check_tissue=False,
   pixel_overlap=0, # default
   prefix="grid/", 
   suffix=".png"

grid_tiles_extractor.locate_tiles(prostate_slide)
```

**Expected behavior**
right

**Instead....**
wrong

**Software (please complete the following information):**
 - OS: macOS
 - Python Version: 3.8

**Additional context**
It has to do with: https://stackoverflow.com/questions/49084846/why-different-size-when-converting-pil-image-to-numpy-array

### PR DESCRIPTION
**Pull request by alessiamarcolini**
fix #184

### CODE REVIEW

File: src/histolab/tiler.py [hunk 1]
```diff
@@ -247,8 +247,8 @@ def _grid_coordinates_from_bbox_coordinates(
 
         for i in range(n_tiles_row):
             for j in range(n_tiles_column):
-                x_ul_lvl = bbox_coordinates.x_ul + tile_w_lvl * j - self.pixel_overlap
-                y_ul_lvl = bbox_coordinates.y_ul + tile_h_lvl * i - self.pixel_overlap
+                x_ul_lvl = bbox_coordinates.x_ul + tile_w_lvl * i - self.pixel_overlap
+                y_ul_lvl = bbox_coordinates.y_ul + tile_h_lvl * j - self.pixel_overlap
```
Comment (ernestoarbitrio) [inline src/histolab/tiler.py:251]: 😭

Comment (alessiamarcolini) [reply src/histolab/tiler.py:251]: I know, kill me now

File: tests/integration/test_tiler.py [hunk 2]
```diff
@@ -3,24 +3,38 @@
...
+    )
+    def it_locates_tiles_on_the_slide(self, fixture_slide, expectation_path, tmpdir):
+        slide = Slide(fixture_slide, os.path.join(tmpdir, "processed"))
         slide.save_scaled_image(10)
         random_tiles_extractor = RandomTiler(
             tile_size=(512, 512), n_tiles=2, level=0, seed=42, check_tissue=False
         )
         expectation = load_expectation(
-            "tiles-location-images/cmu-1-small-region-tiles-location-random",
+            expectation_path,
```
Comment (ernestoarbitrio) [inline tests/integration/test_tiler.py:37]: i'd prefer `expectation` rather than `expectation_path`

File: tests/integration/test_tiler.py [hunk 3]
```diff
@@ -31,16 +45,27 @@ def it_locates_tiles_on_the_slide(self, tmpdir):
...
+        slide = Slide(fixture_slide, os.path.join(tmpdir, "processed"))
         grid_tiles_extractor = GridTiler(
             tile_size=(512, 512),
             level=0,
             check_tissue=False,
         )
-        expectation = load_expectation(
-            "tiles-location-images/cmu-1-small-region-tiles-location-grid", type_="png"
-        )
+        expectation = load_expectation(expectation_path, type_="png")
```
Comment (ernestoarbitrio) [inline tests/integration/test_tiler.py:68]: same here

File: tests/integration/test_tiler.py [hunk 4]
```diff
@@ -49,17 +74,30 @@ def it_locates_tiles_on_the_slide(self, tmpdir):
...
             tile_size=(512, 512),
-            n_tiles=100,
+            n_tiles=2,
             level=0,
-            check_tissue=False,
+            check_tissue=True,
         )
         expectation = load_expectation(
-            "tiles-location-images/cmu-1-small-region-tiles-location-scored",
+            expectation_path,
```
Comment (ernestoarbitrio) [inline tests/integration/test_tiler.py:100]: and here

**Review by ernestoarbitrio — CHANGES_REQUESTED** [review]
some minor comments  ... after that 👍🏽

**Pull request merged**